src/modules/Todo.js

```diff
@@ -0,0 +1,237 @@
+export default class Todo {
+  // todo list arrat to store all todos
+  static todoList = [
+    {
+      description: 'first todo . . . ',
+      completed: false,
+      index: 0,
+    },
+  ];
+
+  // initialize todo values
+  constructor(description, completed) {
+    this.description = description;
+    this.completed = completed;
+    this.index = Todo.todoList.length;
+  }
+
+  // update localstorage for todos
+  static updateLocalstorage = () => {
+    localStorage.setItem('todo', JSON.stringify(Todo.todoList));
+  };
+
+  // add todo into Todo's todoList
+  static setTodo = (description, completed) => {
+    const todo = new Todo(description, completed);
+    const todoList = document.querySelector('.list-container');
+    Todo.todoList.push(todo);
+    Todo.updateUI(todoList);
+    // update localstorage for todos
+    Todo.updateLocalstorage();
+  };
+
+  // update todo as a whole (especially) from localstorage
+  static updateTodo = (currentTodo) => {
+    Todo.todoList = currentTodo;
+    Todo.updateUI(document.querySelector('.list-container'));
+  };
```

Comment:
you are not going to need these updated todo functions because you have not used them anywhere.

Reply:
That's right, 👍 Thank you for picking that out. I almost forgot it: 🤯